packages/deployer/src/infrastructure/serverless.ts

```diff
@@ -0,0 +1,34 @@
+import * as util from 'util';
+import * as child from 'child_process';
+const exec = util.promisify(child.exec);
+import ora from 'ora';
+
+export async function deployAirnode(providerIdShort, region, stage) {
+  const spinner = ora('Deploying the serverless functions').start();
+  try {
+    await deployServerless(providerIdShort, region, stage);
+    spinner.succeed('Deployed the serverless functions');
+  } catch (e) {
+    spinner.fail('Failed deploying the serverless functions');
+    throw e;
+  }
+}
+
+export async function removeAirnode(providerIdShort, region, stage) {
+  const spinner = ora('Removing the serverless functions').start();
+  try {
+    await removeServerless(providerIdShort, region, stage);
+    spinner.succeed('Removed the serverless functions');
+  } catch (e) {
+    spinner.fail('Failed removing the serverless functions.');
+    throw e;
+  }
+}
+
+async function deployServerless(providerIdShort, region, stage) {
+  await exec(`PROVIDER_ID_SHORT=${providerIdShort} REGION=${region} STAGE=${stage} yarn run sls:deploy`);
```

Comment:
This `exec()` business is extremely brittle, but that's not a huge issue when you're running the deployer in a container